content/hi/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,128 @@
+---
+title: सीग्रुप v2 के बारे में  
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+लिनक्स पर, {{< glossary_tooltip text="नियंत्रण समूह" term_id="cgroup" >}} उन संसाधनों को बाधित करते हैं जो प्रक्रियाओं को आवंटित किए जाते हैं।
+
+[पॉड्स और कंटेनरों के लिए संसाधन प्रबंधन](/docs/concepts/configuration/manage-resources-containers/) को लागू करने के लिए क्यूबलेट और अंतर्निहित रनटाइम कंटेनर को सीग्रुप के साथ इंटरफेस करने की आवश्यकता होती है जिसमें कंटेनरीकृत कार्य भार के लिए सीपीयू/मेमोरी अनुरोध और सीमाएं शामिल होती हैं।
+
+Linux में cgroups के दो संस्करण हैं: सीग्रुप v1 और सीग्रुप v2। 
+सीग्रुप v2
+`सीग्रुप` एपीआई की नई पीढ़ी है।
+
+<!-- body -->
+
+
+## सीग्रुप v2 क्या है?? {#cgroup-v2}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
+
+सीग्रुप v2 Linux `सीग्रुप` API का अगला संस्करण है।. 
+सीग्रुप v2 उन्नत संसाधन प्रबंधन के साथ एक एकीकृत नियंत्रण प्रणाली प्रदान करता है
+
+
+सीग्रुप v2, सीग्रुप v1 की तुलना में कई सुधार प्रदान करता है, जैसे कि निम्नलिखित:
+
+- एपीआई में एकल एकीकृत पदानुक्रम डिज़ाइन
+- कंटेनरों को सुरक्षित उपवृक्ष प्रत्यायोजन
+- [प्रेशर स्टॉल सूचना](https://www.kernel.org/doc/html/latest/accounting/psi.html) जैसी नई सुविधाएँ
+- कई संसाधनों में उन्नत संसाधन आवंटन प्रबंधन और अलगाव
+  - विभिन्न प्रकार के मेमोरी आवंटन (नेटवर्क मेमोरी, कर्नेल मेमोरी, आदि) के लिए एकीकृत लेखांकन
+  -पेज कैश राइट बैक जैसे गैर-तत्काल संसाधन परिवर्तनों के लिए लेखांकन
+
+
+कुछ Kubernetes सुविधाएँ उन्नत संसाधन के लिए विशेष रूप से सीग्रुप v2 का उपयोग करती हैं
+प्रबंधन और अलगाव. उदाहरण के लिए, [मेमोरी QoS](/blog/2021/11/26/qos-memory-resources/) सुविधा मेमोरी QoS में सुधार करती है और cgroup v2 प्रिमिटिव पर निर्भर करती है।
+
+## सीग्रुप v2 का उपयोग {#using-cgroupv2}
+
+सीग्रुप v2 का उपयोग करने का अनुशंसित तरीका लिनक्स वितरण का उपयोग करना है
+डिफ़ॉल्ट रूप से सीग्रुप v2 को सक्षम और उपयोग करता है.
+
+यह जाँचने के लिए कि क्या आपका वितरण cgroup v2 का उपयोग करता है, [लिनक्स नोड्स पर cgroup संस्करण की पहचान करें देखें।](#check-cgroup-version)
+### आवश्यकताएं
+
+सीग्रुप v2 की निम्नलिखित आवश्यकताएँ हैं:
+
+* OS वितरण सीग्रुप v2 को सक्षम बनाता है
+* लिनक्स कर्नेल संस्करण 5.8 या उसके बाद का है
+* कंटेनर रनटाइम सीग्रुप v2 का समर्थन करता है। उदाहरण के लिए:
+  * [containerd](https://containerd.io/) v1.4 और बाद का संस्करण
+  * [cri-o](https://cri-o.io/) v1.20 और बाद का संस्करण
+* क्यूबलेट और कंटेनर रनटाइम को [सिस्टमडी सीग्रुप ड्राइवर](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver) का उपयोग करने के लिए कॉन्फ़िगर किया गया है 
+
+### लिनक्स वितरण सीग्रुप v2 समर्थन
+
+
+सीग्रुप v2 का उपयोग करने वाले Linux वितरणों की सूची के लिए,  [सीग्रुप v2 दस्तावेज़](https://github.com/opencontainers/runc/blob/main/docs/cgroup-v2.md) देखें
```

Comment:
cgroup v2 का उपयोग करने वाले Linux वितरणों की सूची के लिए,  [cgroup v2 दस्तावेज़]

According to the changes it would looks good